hudi-flink/src/main/java/org/apache/hudi/operator/InstantGenerateOperator.java

```diff
@@ -222,4 +236,59 @@ public void close() throws Exception {
       fs.close();
     }
   }
+
+  private boolean checkReceivedData(long checkpointId) throws InterruptedException, IOException {
+    int numberOfParallelSubtasks = getRuntimeContext().getNumberOfParallelSubtasks();
+    FileStatus[] fileStatuses = null;
+    Path generatePath = new Path(INSTANT_GENERATE_FOLDER_NAME);
```

Comment:
@loukey-lj how about moving this path to `.hoodie/.aux`

Reply:
agree